docs/themes/htx/layout/partials/toc.ejs

```diff
@@ -28,6 +28,9 @@
       <% if (fileName === 'ml') { %>
           <li><h4>Machine Learning Setup</h4></li>
       <% } %>
+      <% if (fileName === 'webhooks') { %>
+          <li><h4>Integrations</h4></li>
```

Comment:
Is is intentionally structured like this
![image](https://user-images.githubusercontent.com/6087484/129718352-41a2e73c-e2a4-415a-86bb-b024648bd4fc.png)

I feel more like 
- Webhooks
   - Setup
   - Event Reference
   - Custom
- Backend API

Reply:
Yes, I wanted to have the webhook info on separate pages so that the "how to" was separate from the reference information. All the other webhook documentation I looked it is done the same way. Same with the build your own custom webhook.